### ISSUE TITLE
app-antivirus/malheur-0.5.4-r1 fails tests

### ISSUE THREAD

**Issue by asarubbo** [pentoo/pentoo-overlay#757]
https://blogs.gentoo.org/ago/2020/07/04/gentoo-tinderbox/

Issue: app-antivirus/malheur-0.5.4-r1 fails tests.
Discovered on: amd64 (internal ref: pentoo_tinderbox)

At the time of writing, github-cli does not support attachments. I'm using gists but they can be deleted at a certain point. If you want to save them, please create attachments on your own.

build log: https://gist.github.com/3ccc46cb3bc78af1529d0a0a848f7729
1-test-suite.log: https://gist.github.com/9da487f2f38dd0f50e5209581fbda0dd

**Comment by blshkv** [comment]
Yes, yes, we know. Our ebuilds are not perfect, that's why it is still not in Gentoo. Please create a pull request with a fix you care
